enterprise/backend/test/metabase_enterprise/enhancements/models/permissions/block_permissions_test.clj

```diff
@@ -139,7 +152,8 @@
                                       {:schemas :block, :native :write})]
           (is (schema= {:message  #".*DB permissions with a valid combination of values for :native and :schemas.*"
                         s/Keyword s/Any}
-                       (mt/user-http-request :crowberto :put 500 "permissions/graph" new-graph))))))))
+                       (premium-features-test/with-premium-features #{:advanced-permissions}
+                         (mt/user-http-request :crowberto :put 500 "permissions/graph" new-graph)))))))))
```

Comment:
This is not part of this PR but I think we probably should be returning a 400 instead of a 500 here

Reply:
Yeah, let's tackle that in a separate PR.